terms.html

```diff
@@ -123,12 +130,22 @@
 verifiable data registry. For more information, see [[VC-DATA-MODEL]].
   </dd>
 
+  <dt><dfn>DID resolution</dfn></dt>
+
+  <dd>
+The function that takes as its input a <a>DID</a> and a set of resolution
+options and returns a <a>DID document</a> in a conforming representation
```

Comment:
Suggest shifting to use "input metadata" instead of "resolver options" here and throughout. WG should discuss actual terms.

```suggestion
The function that takes as its input a <a>DID</a> and a set of input
metadata and returns a <a>DID document</a> in a conforming representation
```